File: test/fixtures/output/elixir/httpoison/full.ex
HTTPoison.request(
  :post,
  "http://mockbin.com/har?foo=bar&foo=baz&baz=abc&key=value",
  "foo=bar",
  [{"accept", "application/json"}, {"content-type", "application/x-www-form-urlencoded"}],
  hackney: [cookie: "foo=bar; bar=baz"]
)

File: src/targets/node/request/fixtures/full.js
const request = require('request');

const jar = request.jar();
jar.setCookie(request.cookie('foo=bar'), 'http://mockbin.com/har');
jar.setCookie(request.cookie('bar=baz'), 'http://mockbin.com/har');

const options = {
  method: 'POST',
  url: 'http://mockbin.com/har',
  qs: {foo: ['bar', 'baz'], baz: 'abc', key: 'value'},
  headers: {
    accept: 'application/json',
    'content-type': 'application/x-www-form-urlencoded'
  },
  form: {foo: 'bar'},
  jar: jar
};

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});